ts/utils/__tests__/calendar.test.ts

```diff
@@ -0,0 +1,21 @@
+import I18n from "../../i18n";
+import { convertLocalCalendarName } from "../calendar";
+
+describe("Convert calendar title to readable string", () => {
+  const requestedCalendar: string = "calendar_displayname_local";
```

Comment:
We could add another stress test about spacing and lower/upper case
`const requestedCalendar: string = "caLeNdar_DisplaynAme_local   ";`

Reply:
sounds good! it should return a positive case such as the base test!